project-koku/koku
# add uuid to infrastructure in /sources/

This PR adds the UUID to the infrastructure field in the `/sources/` endpoint based on conversations with Dan from UI.

Testing:

- create and load test customer data
- go to: http://localhost:8000/api/cost-management/v1/sources/
- Look at the ocp sources and see the ones with infrastructure info include the type and UUID that matches the aws/azure source UUID. Sources without infrastructure have an empty dictionary.
```
====================================================================== short test summary info =======================================================================
FAILED lib64/python3.6/site-packages/iqe_cost_management/tests/rest_api/v1/test__ocp_cost_reports.py::test_api_ocp_cost_order_by_name_pagination - AssertionError: ...
FAILED lib64/python3.6/site-packages/iqe_cost_management/tests/rest_api/v1/test_azure_cost_reports.py::test_api_azure_cost_tag_validate_keys_visible - AssertionErr...
FAILED lib64/python3.6/site-packages/iqe_cost_management/tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_order_by_name_pagination - AssertionError: F...
============================================== 3 failed, 5831 passed, 201 skipped, 648 deselected in 3783.20s (1:03:03) ==============================================
```
